test/uk/gov/hmrc/ngrpropertylinkingfrontend/controllers/CurrentRatepayerControllerSpec.scala

```diff
@@ -27,18 +28,22 @@ import uk.gov.hmrc.auth.core.Nino
 import uk.gov.hmrc.http.HeaderNames
 import uk.gov.hmrc.ngrpropertylinkingfrontend.helpers.ControllerSpecSupport
 import uk.gov.hmrc.ngrpropertylinkingfrontend.models.registration.CredId
-import uk.gov.hmrc.ngrpropertylinkingfrontend.models.{AuthenticatedUserRequest, PropertyLinkingUserAnswers}
+import uk.gov.hmrc.ngrpropertylinkingfrontend.models.{AuthenticatedUserRequest, CurrentRatepayer, PropertyLinkingUserAnswers}
+import uk.gov.hmrc.ngrpropertylinkingfrontend.views.html.components.DateTextFields
 import uk.gov.hmrc.ngrpropertylinkingfrontend.views.html.{AddPropertyToYourAccountView, CurrentRatepayerView}
 
+import java.time.LocalDate
 import scala.concurrent.Future
 
 class CurrentRatepayerControllerSpec extends ControllerSpecSupport with DefaultAwaitTimeout {
   implicit val requestHeader: RequestHeader = mock[RequestHeader]
   lazy val currentRatepayerView: CurrentRatepayerView = inject[CurrentRatepayerView]
+  lazy val dateTextFields: DateTextFields = inject[DateTextFields]
```

Comment:
No point in this and the field above being lazy as they are always instantiated and always used